app/config/AppConfig.scala

```diff
@@ -46,7 +47,9 @@ class AppConfigImpl @Inject() (val config: Configuration, servicesConfig: Servic
   private val contactBaseUrl = servicesConfig.baseUrl("contact-frontend")
 
   private val assetsUrl = config.get[String]("assets.url")
-  private val serviceIdentifier = "MyService"
+  private val serviceIdentifier = "vegf"
```

Comment:
Would remove private here as it may need to be referred to outside of the Config class